### ISSUE TITLE
[Suggestion] Autonomous Activator-like machine

### ISSUE THREAD

**Issue by ToadsworthLP** [BluSunrize/ImmersiveEngineering#1514]
Could you add a small (2-3 blocks) multiblock machine that acts like the autonomous activator from TE? This would open up so many possibilities, for example, a cobblestone generator (by equipping it with a pickaxe) or a tree farm (equipping it with a Lumberaxe from TC).
PS: Sorry that I'm doing the third "suggestion" in such a short time, maybe I do GitHub wrong.
PPS: TE was open-sourced a few months ago, this could help you to implement it: https://github.com/CoFH/ThermalExpansion/blob/master/src/main/java/cofh/thermalexpansion/block/device/TileActivator.java

**Comment by malte0811** [comment]
Regarding the PR: I am still working on changing the excavator's bucket wheel to the new rendering method (more precisely, I need to find a mineral vein for testing... and the mineral command is broken, probably because I made client side commands work, so I need to fix that as well), once I push that commit, IE is ready for a release in my opinion.

**Comment by ToadsworthLP** [comment]
@malte0811 By the way, what do you think of my suggestion itself? Would you like that too?

**Comment by KnightMiner** [comment]
There are autoclickerd in other mods, notably Extra Utilities, so it really depends on whether thematically it would fit, which I personally don't think so as something reseming a claw for "clicking" would look out of place

**Comment by ToadsworthLP** [comment]
@KnightMiner for the thematic, it would fit if it had a nice model. Sadly, as far as I know, Damien Hazard (the mod's artist), is currently busy, so he won't be able to make a model. But @BluSunrize can implement it in the meantime, and make it a config option as long as is doesn't have a pretty model. Maybe some fans make thier own model?

**Comment by mindforger** [comment]
>There are autoclickerd in other mods, notably Extra Utilities

wait what? where? wich block?

**Comment by BluSunrize** [comment]
@ToadsworthLP you make it sound like I can't model, lol! Damien's definitely better than me with textures and stuff, but I did do quite a bit of the models =P

I have another issue with this suggestion however. I think that a multiblock autoclicker would be incredibly over encumbering. People don't want a 3x3x3 to pick a single berry bush. If this machine becomes a thing it'd be 1x1 or 1x2.

Secondly: Fake leftclicking is a pain in the ass. Fake rightclicking is already annoying because it requires fake players and stuff, but leftclicking is, as far as I remember, even worse because of blockbreak progress and things like that.

Thirdly, I do hvae /plans/ regarding a machine similar to this. I have had plans for basically a whole new tech-layer in IE for months now, I just haven't gotten around to working on it. Be patient folks, I'm sure I'll get around to it at some point ^_^

**Comment by ToadsworthLP** [comment]
Yay ^^

**Comment by ToadsworthLP** [comment]
@BluSunrize Should I leave this issue open, so you remember it later?

**Comment by BluSunrize** [comment]
Yeah, sure, we'll leave it up. I do cleanups of my git every 3 months or so. It can stay till then.
